configs/body/3d_kpt_mview_rgb_img/README.md

```diff
@@ -0,0 +1,8 @@
+# Multi-view 3D Human Body Pose Estimation
+
+3D pose estimation is the prediction of X, Y, Z coordinates of human body joints from multi-view RGB images.
```

Comment:
Multi-view 3D human body pose estimation targets at predicting the X, Y, Z coordinates of human body joints from multi-view RGB images.